WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift

```diff
@@ -51,6 +43,38 @@ final class DashboardQuickStartCardCell: UICollectionViewCell, Reusable, BlogDas
                           properties: ["type": DashboardCard.quickStart.rawValue],
                           blog: blog)
     }
+
+    private func configureCardFrameView(for blog: Blog) {
+        switch blog.quickStartType {
```

Comment:
Same comment here for handling `undefined` as if it's `newSite`

Reply:
We should probably do the same for `String.title(for:)`. This is probably an existing bug 🙈